authors/apps/authentication/views.py

```diff
@@ -1,17 +1,28 @@
+import os
+import jwt
+from django.urls import reverse
+from django.contrib.sites.shortcuts import get_current_site
+from django.shortcuts import get_object_or_404
+from django.core.mail import send_mail
 from rest_framework import status
-from rest_framework.generics import RetrieveUpdateAPIView, GenericAPIView
+from rest_framework.generics import RetrieveUpdateAPIView, UpdateAPIView, GenericAPIView
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
-
+from rest_framework import serializers
+from authors.apps.authentication.models import User
 from .renderers import UserJSONRenderer
 from .serializers import (
-    LoginSerializer, RegistrationSerializer, UserSerializer
+    LoginSerializer,
+    RegistrationSerializer,
+    UserSerializer,
+    PasswordResetSerializser,
+    PasswordResetEmailSerializer
 )
 from .validation import validate_registration
 
 
-class RegistrationAPIView(GenericAPIView):
+class RegistrationAPIView(APIView):
```

Comment:
Hey @anyatibrian, thanks for the amazing work done so far! Was just wondering why you had to change from GenericAPIView to APIView.